x-pack/plugins/upgrade_assistant/server/routes/cluster_checkup.test.ts

```diff
@@ -3,8 +3,8 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import { kibanaResponseFactory } from 'src/core/server';
-import { createMockRouter, MockRouter } from './__mocks__/routes.mock';
+import { kibanaResponseFactory } from '../../../../../src/core/server';
```

Comment:
So now I am confused about all my comments about using an `alias`. This change is because we don't have an alias setup?